mllib/src/main/scala/org/apache/spark/ml/feature/ChiSqSelector.scala

```diff
@@ -119,6 +121,9 @@ final class ChiSqSelectorModel private[ml] (
   /** list of indices to select (filter). Must be ordered asc */
   val selectedFeatures: Array[Int] = chiSqSelector.selectedFeatures
 
+  /** Java-friendly version of [[selectedFeatures]]. */
+  def javaSelectedFeatures: java.util.List[Int] = selectedFeatures.toSeq.asJava
```

Comment:
This isn't needed for Java, so I'd make it a private API.  But hopefully we can remove this altogether once [https://github.com/apache/spark/pull/10724] gets merged.